notebooks/GWAS_1000_genomes_p1/main.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Notebook author: Beth Sheets
+# Herzog version: Ash O'Farrell
+
+# The notebook didn't have a markdown title at the time but I've dubbed it Phenotype Consolidator.
+# The name of the actual file is 1-Prepare-Gen3-Data-For-Exploration
+
+import os
+import herzog
+
+# Mock the environment
+os.environ['WORKSPACE_NAME'] = "CICD TESTER - PUBLIC DATA ONLY - BDC GWAS 1000G Tutorial"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-7d1224a1-1770-4db9-9904-137efcd412c5"
+os.environ['GOOGLE_PROJECT'] = "anvil-stage-demo"
+
+with herzog.Cell("markdown"):
+    """
+    # Phenotype Consolidater
+    *version: 2.0*
+
+    ## Notebook Overview
+
+    This notebook consolidates the phentotypic data in tables imported to Terra from Gen3 into one consolidated table of metadata with familiar subject IDs for downstream analysis. It uses functions defined in the terra-util notebook, and can be adapted to consolidate different metadata.
```

Comment:
double check that we reference "terra_data_table_util" correctly throughout the notebook. I know MB changed the name of it awhile back. Here, we say "terra_util" which could be confusing.